spec/services/ai_conversations/stream_service_spec.rb

```diff
@@ -2,85 +2,170 @@
 
 require "rails_helper"
 
-RSpec.describe AiConversations::StreamService do
+describe AiConversations::StreamService, type: :service do
```

Comment:
I think this is no more mandatory, the type is inferred automatically from the file path